ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,52 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+SFPC:
```

Comment:
Can we instead use a combination of `SSPM` and `SFC`, along with maybe `SFSS`?

Reply:
We can if i add the rotational and linear velocity fields onto one of the suggested types?